src/eos-updater-ctl

```diff
@@ -92,12 +96,8 @@ def name_appeared_cb(connection, name, name_owner):
                                    'com.endlessm.Updater',
                                    None)  # cancellable
 
-    proxy.connect('g-properties-changed',
-                  lambda proxy, changed, invalidated, user_data: \
-                      dump_daemon_properties(proxy), None)
-    proxy.connect('g-signal',
-                  lambda proxy, sender, signal, parameters, user_data: \
-                      signal_emitted(proxy, signal, parameters), None)
+    proxy.connect('g-properties-changed', g_properties_changed_cb)
+    proxy.connect('g-signal', signal_emitted, None)
```

Comment:
OK, but shouldn’t the function names be prefixed with `__` to make them private?

Reply:
This script can't be imported as a module because its name contains hyphens, so by definition all top-level functions are private…

Reply:
TIL

Reply:
Well, "can't" is a strong word.

```python
In [1]: import eos-updater-ctl
  File "<ipython-input-1-89bee1140f50>", line 1
    import eos-updater-ctl
              ^
SyntaxError: invalid syntax


In [2]: import imp

In [3]: eos_updater_ctl = imp.load_source('m', 'eos-updater-ctl')

In [4]: dir(eos_updater_ctl)
Out[4]: 
['EOS_UPDATER_CONFS',
 'EOS_UPDATE_SERVER_CONFS',
 'GLib',
 'Gio',
 'METHODS',
 'UPDATER_STATES',
 '__builtins__',
 '__doc__',
 '__file__',
 '__loader__',
 '__name__',
 '__package__',
 '__spec__',
 'argparse',
 'command_apply',
 'command_dbus',
 'command_fetch',
 'command_monitor',
 'command_poll',
 'command_server',
 'command_update',
 'current_proxy',
 'dump_daemon_properties',
 'g_properties_changed_cb',
 'main',
 'name_appeared_cb',
 'name_vanished_cb',
 'nonnegative_int',
 'os',
 'signal_emitted',
 'sys']
```

I guess I mean they are all conceptually private. I don't really like the `__` prefix at the best of times but it really seems like overkill in a standalone script.